jlh/JonFirstApp/app/src/main/res/layout/activity_display_message.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We should try to use a `LinearLayout` as the root layout as much as possible. The reasoning behind this is that when the interface is redrawn (for ex. orientation change), the `RelativeLayout` performs extra calculations for placing the views in the right order. You can find more info [here](https://developer.android.com/guide/topics/ui/declaring-layout.html#CommonLayouts). In this simple case we don't have multiple views that need to be placed relative to one another.